src/modules/Search/Search.js

```diff
@@ -166,6 +166,11 @@ export default class Search extends Component {
     /** A search can have its results take up the width of its container. */
     fluid: PropTypes.bool,
 
+    /** A search input can take up the width of its container . */
+    input: PropTypes.shape({
```

Comment:
This will be a shorthand value, `customPropTypes.itemShorthand`.  Our shorthand prop type allows strings, numbers, props objects, and other elements.  These shorthand prop values are used with our shorthand `create()` methods, see below.